### DISCUSSION TITLE
Kedro-Viz 5.0.0 is out!

### DISCUSSION THREAD

**Discussion by tynandebold** [kedro-org/kedro#1774] [Announcements]
### We're excited to announce that Kedro-Viz 5.0.0 is out! 💥 

[Kedro-Viz](https://github.com/kedro-org/kedro-viz) is an interactive development tool for building data science pipelines with [Kedro](https://github.com/kedro-org/kedro). It also allows users to view and compare different runs in their Kedro project.

This new major-release version brings a great new feature and the removal of support for an older version of Kedro. Have a look at the [full release notes](https://github.com/kedro-org/kedro-viz/releases/tag/v5.0.0) for everything new. Check out our [hosted demo](https://demo.kedro.org/).

**What can you expect in this release?**
- Support for visualising Matplotlib and Plotly plots in experiment tracking :bar_chart:
- Remove support for Kedro v16 :broom:

:warning: Note this last point is a breaking change! If you're using Kedro v17 or v18, please upgrade to continue getting the best out of Kedro-Viz. :warning:

**How do I get access to the latest release?**
- Get the latest version of Kedro-Viz with `pip install kedro-viz==5.0.0` (Python) or with `npm install @quantumblack/kedro-viz@latest` (React)

**What's next?**
- Improvements on [Experiment Tracking](https://kedro.readthedocs.io/en/stable/tutorial/set_up_experiment_tracking.html), like being able to track and plot metrics over time.
- And follow [our roadmap](https://github.com/orgs/kedro-org/projects/3/views/1) for more updates.

**Anything else?**
We have launched [a Kedro Slack organisation, and you should join it](https://join.slack.com/t/kedro-org/shared_invite/zt-1czm8jjjq-axjiZxUzOrc26nx2GiuWTA).

Until next time,
The Kedro Team :purple_heart:

**Comment by andrewcstewart** [comment]
Slack invite is no longer active, fyi!

**Reply by antonymilne** [reply]
Thanks for letting us know! Please can you try this one? https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw
